### PR TITLE
feat: convert routes to classes with decorators using typescript-rest

### PR DESCRIPTION
**Pull request by bertyhell**

### CODE REVIEW

File: src/helpers/errorHandler.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { pathOr, startsWith } from 'ramda';
+import * as _ from 'lodash';
```
Comment (e2o) [inline src/helpers/errorHandler.ts]: import { get } from "lodash"

File: src/middleware/data/copy.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import { Response, NextFunction } from 'express';
 import { DataRequest } from '../middleware.types';
-import { clone } from 'ramda';
+import * as _ from 'lodash';
```
Comment (e2o) [inline src/middleware/data/copy.ts]: import { cloneDeep } from "lodash"

File: src/helpers/errorHandler.ts (hunk 1 above)
Comment (bertyhell) [reply src/helpers/errorHandler.ts]: do we care on the backend?
we gaan toch geen treeshaking doen

File: src/app.ts [hunk 3]
```diff
@@ -1,22 +1,34 @@
...
 import express, { Application } from 'express';
 import global from './middleware/global';
 import errorHandler from './middleware/errorHandler';
 import { AddressInfo } from 'net';
-import routes from './modules/routes';
 import { Logger } from './helpers/logger';
+import { Server } from 'typescript-rest';
+import StatusRoute from './modules/status/route';
+import SearchRoute from './modules/search/route';
+import cors from 'cors';
```
Comment (e2o) [inline src/app.ts]: Groepeer en orden uw imports hier:

```
import x from "externalX"
import y from "externalY"

import xMiddleware from "./middleware/x"
import yMiddleware from "./middleware/y"
import zMiddleware from "./middleware/z"

import xHelper from "./helpers/x"
...

```

File: src/helpers/validation/index.ts [hunk 4]
```diff
@@ -1,9 +1,8 @@
 import Joi from 'joi';
-import { curry } from 'ramda';
 import { ValidationErr } from '../validationError';
 import presets from './presets';
 
-const validator = curry((preset, onFailError, source) => {
+const validator = (preset, onFailError, source) => {
```
Comment (e2o) [inline src/helpers/validation/index.ts]: Curry moet hier wel behouden worden, omzetten naar die van lodash?

File: src/app.ts (hunk 3 above)
Comment (bertyhell) [reply src/app.ts]: fixed in most recent PR

File: src/helpers/validation/index.ts (hunk 4 above)
Comment (bertyhell) [reply src/helpers/validation/index.ts]: fixed in most recent PR

File: src/middleware/data/copy.ts (hunk 2 above)
Comment (bertyhell) [reply src/middleware/data/copy.ts]: switched to `import _ from 'lodash';`

File: src/helpers/errorHandler.ts (hunk 1 above)
Comment (bertyhell) [reply src/helpers/errorHandler.ts]: switched to `import _ from 'lodash';`

**Pull request closed without merging**